robottelo/config/__init__.py

```diff
@@ -1,4 +1,48 @@
-from robottelo.config.base import Settings
+import logging
 
-#: A :class:`Settings` object.
-settings = Settings()
+from dynaconf import LazySettings
+from dynaconf.validator import ValidationError
+
+from .validators import validators
+from robottelo.config.base import ImproperlyConfigured
+from robottelo.config.base import Settings as LegacySettings
+from robottelo.config.proxy import ConfigNodeWrapper
+from robottelo.config.proxy import ConfigProxy
+
+logger = logging.getLogger('robottelo.config')
+
+legacy_settings = LegacySettings()
+
+dynaconf_settings = LazySettings(
+    envvar_prefix="SATQE",
+    core_loaders=["YAML"],
+    settings_file="settings.yaml",
+    preload=["conf/*.yaml"],
+    includes=["settings.local.yaml", ".secrets.yaml", ".secrets_*.yaml"],
+    envless_mode=True,
+    lowercase_read=True,
+)
+dynaconf_settings.validators.register(**validators)
+
+try:
+    legacy_settings.configure()
+except ImproperlyConfigured:
+    logger.error(
+        (
+            "Legacy Robottelo settings configure() failed, most likely required "
+            "configuration option is not provided. Continuing for the sake of unit tests"
+        ),
+        exc_info=True,
+    )
+
+try:
+    dynaconf_settings.validators.validate()
+except ValidationError:
+    logger.error(
+        "Dynaconf validation failed, continuing for the sake of unit tests", exc_info=True
+    )
+
+settings_proxy = ConfigProxy()
```

Comment:
1. As Robottelo user and developer, you are never expected to interact with that object and class name directly. The whole point is that it's transparent for users within application.
2. ["Proxy" is established name of the pattern](https://en.wikipedia.org/wiki/Proxy_pattern). "Proxy" is used as synonym for "substitute" in many different contexts, also outside of programming. I can understand how it can be confusing, but that's just effect of term being overloaded.

What name would you propose? Would `SettingsFacade` be better?

Reply:
As you wish, renamed in 87d37aee9a7457d632bba18802f55207ba0a11cc :
* `ConfigProxy` -> `SettingsFacade`
* `ConfigNodeWrapper` -> `SettingsNodeWrapper`